data/yaml/name_variants.yaml

```diff
@@ -3629,7 +3629,9 @@
 - canonical: {first: Ahmed, last: Hassan}
   variants:
   - {first: Ahmed Hassan, last: Awadallah}
-- canonical: {first: Hany, last: Hassan}
+- canonical: {first: Hany, last: Hassan Awadalla}
+  variants:
+  - {first: Hany, last: Hassan}
   id: hany-hassan
```

Comment:
@mjpost Because it says so right here :)

Reply:
(The ID is used as the author page slug, if it is given)

Reply:
Ah, right, I thought it was built from the canonical name. Thank you!